core/src/main/java/org/apache/carbondata/core/profiler/TablePruningInfo.java

```diff
@@ -99,4 +107,44 @@ public String toString() {
     }
     return builder.toString();
   }
+
+  /**
+   * when CACHE_LEVEL = BLOCK or carbon data file is LegacyStore
+   * only show pruned result size of datamaps in block/blocklet level
+   */
+  private String getHitInfoAfterPruning() {
+    StringBuilder builder = new StringBuilder();
+    builder
+        .append(" - total blocks: ").append(totalBlocklets).append("\n")
+        .append(" - filter: ").append(filterStatement).append("\n");
+    if (defaultDataMap != null) {
+      builder
+          .append(" - pruned by Main DataMap").append("\n")
+          .append("    - hit blocks: ").append(numBlockletsAfterDefaultPruning).append("\n");
```

Comment:
It is better to unified with "blocklet case", you can change to use "hit blocklets" in `getSkipBlockletInfoAfterPruning`